### ISSUE TITLE
Change load_retail_to_accessor once Accessor implementation is complete

### ISSUE THREAD

**Issue by gsheni** [alteryx/woodwork#557]
- Once we have confirmed parity with DataTable, we can rename `load_retail_to_accessor` function to `load_retail`

**Comment by gsheni** [comment]
Duplicate of https://github.com/alteryx/woodwork/issues/495#issuecomment-775296415
